pkgs/development/tools/amazon-qldb-shell/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, clang, cmake, fetchFromGitHub, llvmPackages, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "amazon-qldb-shell";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "awslabs";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Pnm1HxEjjNKpS3tTymtOXxUF7EEnWM+7WBsqeaG8seA=";
+  };
+
+  nativeBuildInputs = [ clang cmake ];
+  buildInputs = [ llvmPackages.libclang ];
+
+  cargoSha256 = "sha256-EUqGSKcGnhrdLn8ystaLkkR31RjEvjW6vRzKPMK77e8=";
+
+  LIBCLANG_PATH = "${llvmPackages.libclang.lib}/lib";
+
+  doInstallCheck = true;
+
+  installCheckPhase = ''
+    $out/bin/qldb --help
+    $out/bin/qldb --version | grep 'qldb ${version}'
+  '';
+
+  meta = {
+    description = "An interface to send PartiQL statements to Amazon Quantum Ledger Database (QLDB)";
+    homepage = "https://github.com/awslabs/amazon-qldb-shell";
+    license = lib.licenses.asl20;
+    maintainers = [ lib.maintainers.terlar ];
+  };
```

Comment:
```suggestion
  meta = with lib; {
    description = "An interface to send PartiQL statements to Amazon Quantum Ledger Database (QLDB)";
    homepage = "https://github.com/awslabs/amazon-qldb-shell";
    license = licenses.asl20;
    maintainers = [ maintainers.terlar ];
  };
```

Reply:
I was trying to avoid `with` due to performance implications. Not sure how much this affects the evaluation though, just remember seeing that across the whole nixpkgs repo `with` added some percentage of time spent on evaluation.

I could switch to use `with` in order to align with the conventions.

Reply:
There luckily isn't a noticable difference, `lib` isn't that big compared to nixpkgs.